crates/net/network/src/metrics.rs

```diff
@@ -118,7 +118,8 @@ pub struct TransactionsManagerMetrics {
     /* ================ POOL IMPORTS ================ */
     /// Number of transactions about to be imported into the pool.
     pub(crate) pending_pool_imports: Gauge,
-    /// Total number of bad imports.
+    /// Total number of bad imports, imports that fail because a transaction is maliciously
+    /// engineered.
```

Comment:
it doesn't have to be malicious necessarily, this is more like transactions that can never be included, because they violate consensus rules or have unreasonable (malicious) parameters

Reply:
alright, can remove it. was referring to the original comment.
```
otherwise we penalize the peer that sent the bad
// transaction with the assumption that the peer should have
// known that this transaction is bad.
```